Bottom: 17acc4804c3fe5e1aa4bc3b82781d076278d67b2
Top:    7afbdcbe4c47deb82be5d57c10b341324d42a64b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-26 16:13:04 +0000

WAS: Use motion hints when processing pointer motion over the drawing area

Not sure what the remaining changes here are for!


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index cd9236b..3aea36e 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -755,7 +755,7 @@ ghid_show_crosshair (gboolean show)
 
   glBegin (GL_LINES);
 
-#if 1
+#if 0
   if (x_prev >= 0)
     draw_crosshair (x_prev, y_prev);
 #endif
@@ -768,7 +768,7 @@ ghid_show_crosshair (gboolean show)
 
   glBegin (GL_QUADS);
 
-#if 1
+#if 0
   if (x_prev >= 0 && draw_markers_prev)
     {
       glVertex2i (0,                  y_prev - VCD);
